diff --git a/tools/Makefile b/tools/Makefile
index 4d91e7a..ecc5503 100644
--- a/tools/Makefile
+++ b/tools/Makefile
@@ -19,7 +19,7 @@ LINUXINCLUDE := -I$(src)/../driver
 KBUILD_CFLAGS := -g -O3 -DLIBEXECDIR=\"$(libexecdir)\" \
 	-Wall -Wextra -Wmissing-declarations -Wmissing-prototypes -Werror \
 	-DJAILHOUSE_VERSION=\"$(shell cat $(src)/../VERSION)\" $(EXTRA_CFLAGS)
-LDFLAGS :=
+#LDFLAGS := $(EXTRA_LDLAGS)
 
 BINARIES := jailhouse
 HELPERS := \
