Index: git/tools/Makefile
===================================================================
--- git.orig/tools/Makefile
+++ git/tools/Makefile
@@ -23,7 +23,7 @@ KBUILD_CFLAGS := -g -O3 -DLIBEXECDIR=\"$
 	-D__LINUX_COMPILER_TYPES_H \
 	-DJAILHOUSE_VERSION=\"$(shell cat $(src)/../VERSION)\" $(EXTRA_CFLAGS)
 # prior to 4.19
-LDFLAGS :=
+#LDFLAGS :=
 # since 4.19
 KBUILD_LDFLAGS :=
 
@@ -101,7 +101,7 @@ endef
 targets += jailhouse.o
 
 $(obj)/jailhouse: $(obj)/jailhouse.o
-	$(call if_changed,ld)
+	$(LD) $(LDFLAGS) -o $@ $^
 
 CFLAGS_jailhouse-gcov-extract.o	:= -I$(src)/../hypervisor/include \
 	-I$(src)/../hypervisor/arch/$(SRCARCH)/include
@@ -112,10 +112,11 @@ targets += jailhouse-gcov-extract.o
 always += jailhouse-gcov-extract
 
 $(obj)/jailhouse-gcov-extract: $(obj)/jailhouse-gcov-extract.o
-	$(call if_changed,ld)
+	$(LD) $(LDFLAGS) -o $@ $^
 
 $(obj)/jailhouse-config-collect: $(src)/jailhouse-config-create $(src)/jailhouse-config-collect.tmpl
-	$(call if_changed,gen_collect)
+	$< -g $@; \
+	chmod +x $@
 
 targets += $(MAN8_PAGES)
 always +=  $(MAN8_PAGES)
