Index: git/tools/Makefile
===================================================================
--- git.orig/tools/Makefile
+++ git/tools/Makefile
@@ -19,7 +19,7 @@ LINUXINCLUDE := -I$(src)/../driver
 KBUILD_CFLAGS := -g -O3 -DLIBEXECDIR=\"$(libexecdir)\" \
 	-Wall -Wextra -Wmissing-declarations -Wmissing-prototypes -Werror \
 	-DJAILHOUSE_VERSION=\"$(shell cat $(src)/../VERSION)\" $(EXTRA_CFLAGS)
-LDFLAGS :=
+
 GCOV_PROFILE := n
 
 BINARIES := jailhouse
@@ -57,7 +57,7 @@ endef
 targets += jailhouse.o
 
 $(obj)/jailhouse: $(obj)/jailhouse.o
-	$(call if_changed,ld)
+	$(LD) $(LDFLAGS) -o $@ $^
 
 CFLAGS_jailhouse-gcov-extract.o	:= -I$(src)/../hypervisor/include \
 	-I$(src)/../hypervisor/arch/$(SRCARCH)/include
@@ -68,10 +68,11 @@ targets += jailhouse-gcov-extract.o
 always += jailhouse-gcov-extract
 
 $(obj)/jailhouse-gcov-extract: $(obj)/jailhouse-gcov-extract.o
-	$(call if_changed,ld)
+	$(LD) $(LDFLAGS) -o $@ $^
 
 $(obj)/jailhouse-config-collect: $(src)/jailhouse-config-create $(src)/jailhouse-config-collect.tmpl
-	$(call if_changed,gen_collect)
+	$< -g $@; \
+	chmod +x $@
 
 install-bin: $(BINARIES) $(DESTDIR)$(sbindir)
 	$(INSTALL_PROGRAM) $^
